action.yml

```diff
@@ -22,13 +22,13 @@ outputs:
     description: 'Build status'
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  image: 'docker://ghcr.io/calance-us/build-jenkins-job-image:v0.0.0-rc0'
```

Comment:
this version should not be v0.0.0-rc0 it should be a production version. 1.0.0 or something like that.

Reply:
done